src/Alert/Alert.js

```diff
@@ -1,174 +1,259 @@
 import React, { useState } from 'react'
-import PropTypes from 'prop-types'
-import { DismissButton } from './DismissButton'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import {
-  faBell,
-  faCheckCircle,
-  faQuestionCircle,
-  faExclamationTriangle,
-  faTimesCircle
-} from '@fortawesome/free-solid-svg-icons'
-import styles from './alert.module.css'
-
-// Default look and feel
-// /////////////////////////////////////////////////////////////////////////////
-const defaultClasses = {
-  wrapper: 'su-alert',
-  dismissButtonWrapper: [
-    'su-order-3',
-    'su-rs-m-l-1',
-    'su-h-full',
-    'su-items-end',
-    'su-flex-shrink',
-    'su-text-right',
-    'su-w-full',
-    'sm:su-w-auto'
-  ].join(' '),
-  dismissButton: '',
-  headerWrapper: [
-    'su-order-1',
-    'su-rs-m-r-1',
-    'su-flex-shrink',
-    'su-mb-4',
-    'xs:su-w-full',
-    'lg:su-w-max'
-  ].join(' '),
-  label: [
-    'su-inline-block',
-    'su-uppercase',
-    'su-font-semibold',
-    'su-text-170rem',
-    'su-h-full',
-    styles.label
-  ].join(' '),
-  bodyWrapper: styles.alertBodyWrapperDark,
-  icon: faBell,
-  iconClass: 'su-mr-2 su-inline-block su-max-w-xs',
-  bodyHeading: '',
-  footerWrapper: 'su-rs-m-t-0'
-}
-
-// Variant styles.
-// /////////////////////////////////////////////////////////////////////////////
-const variants = {
-  success: {
-    wrapper: 'su-alert su-bg-palo-verde',
-    bodyWrapper: styles.alertBodyWrapper,
-    headerWrapper: defaultClasses.headerWrapper + ' su-text-white',
-    footerWrapper: defaultClasses.footerWrapper + ' su-text-white',
-    icon: faCheckCircle
-  },
-  warning: {
-    wrapper: 'su-alert su-bg-illuminating-dark',
-    bodyWrapper: styles.alertBodyWrapperDark,
-    icon: faExclamationTriangle
-  },
-  error: {
-    wrapper: 'su-alert su-bg-digital-red',
-    bodyWrapper: styles.alertBodyWrapper,
-    headerWrapper: defaultClasses.headerWrapper + ' su-text-white',
-    footerWrapper: defaultClasses.footerWrapper + ' su-text-white',
-    icon: faTimesCircle
-  },
-  info: {
-    wrapper: 'su-alert su-bg-bright-blue',
-    bodyWrapper: styles.alertBodyWrapper,
-    headerWrapper: defaultClasses.headerWrapper + ' su-text-white',
-    footerWrapper: defaultClasses.footerWrapper + ' su-text-white',
-    icon: faQuestionCircle
-  }
-}
+import propTypes from 'prop-types'
+import { alertTypes, lightText, darkText } from './Alert.levers'
+import { Button } from '../Button/Button'
```

Comment:
The react-aria library button was a hidden button for screen readers only so I did not use that one. We needed a button component so I started one.